scripts/queries/benchmark_queries/build_and_benchmark_bluesky_ordered.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Work at the root directory
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)